mcp/mcp-cli-api/src/main/resources/META-INF/smithy/mcp-cli-api.smithy

```diff
@@ -44,6 +44,9 @@ structure TelemetryData {
     @default({})
     timings: Timings
 
+    @required
+    exitCode: Integer
```

Comment:
`PrimitiveInteger = 0` to avoid a box

Reply:
Huh, I guess it does this only in smithy-java.

Reply:
Nope :(

Reply:
https://github.com/smithy-lang/smithy-java/pull/819